app/styles/ember-appuniversum.scss

```diff
@@ -15,100 +15,16 @@
  */
 
 
-// VARIABLES
-@import "appuniversum/s-colors.scss";
-@import "appuniversum/s-global";
-@import "appuniversum/s-utilities";
-@import "appuniversum/s-root";
-
-
-// MIXINS
-@import "appuniversum/t-font-size";
-@import "appuniversum/t-sass-mq";
-
-
-// GENERIC
-@import "appuniversum/g-reset";
-@import "appuniversum/g-box-sizing";
-@import "appuniversum/g-font";
-
-
-// ELEMENTS
-@import "appuniversum/e-page";
-
-
-// OBJECTS
-@import "appuniversum/o-box";
-@import "appuniversum/o-flow";
-@import "appuniversum/o-grid";
-@import "appuniversum/o-layout";
-@import "appuniversum/o-region";
-
-
-// COMPONENTS
-@import "ember-appuniversum/c-accordion";
-@import "ember-appuniversum/c-alert";
-@import "ember-appuniversum/c-app";
-@import "ember-appuniversum/c-badge";
-@import "ember-appuniversum/c-body-container";
-@import "ember-appuniversum/c-brand";
-@import "ember-appuniversum/c-button";
-@import "ember-appuniversum/c-button-group";
-@import "ember-appuniversum/c-card";
-@import "ember-appuniversum/c-content";
-@import "ember-appuniversum/c-content-header";
-@import "ember-appuniversum/c-control";
-@import "ember-appuniversum/c-data-table";
-@import "ember-appuniversum/c-dropdown";
-@import "ember-appuniversum/c-file-upload";
-@import "ember-appuniversum/c-form";
-@import "ember-appuniversum/c-heading";
-@import "ember-appuniversum/c-help-text";
-@import "ember-appuniversum/c-hr";
-@import "ember-appuniversum/c-icon";
-@import "ember-appuniversum/c-input";
-@import "ember-appuniversum/c-label";
-@import "ember-appuniversum/c-link";
-@import "ember-appuniversum/c-list-horizontal";
-@import "ember-appuniversum/c-list-navigation";
-@import "ember-appuniversum/c-loader";
-@import "ember-appuniversum/c-pagination";
-@import "ember-appuniversum/c-pill";
-@import "ember-appuniversum/c-main-container";
-@import "ember-appuniversum/c-main-header";
-@import "ember-appuniversum/c-main-footer";
-@import "ember-appuniversum/c-modal";
-@import "ember-appuniversum/c-sidebar";
-@import "ember-appuniversum/c-sidebar-action";
-@import "ember-appuniversum/c-table";
-@import "ember-appuniversum/c-textarea";
-@import "ember-appuniversum/c-timepicker";
-@import "ember-appuniversum/c-toolbar";
-@import "ember-appuniversum/c-toggle-switch";
-
-
-// PLUGINS
-@import "ember-appuniversum/p-ember-power-select";
-@import "ember-appuniversum/p-duet-datepicker";
-
-
-// UTILITIES
-@import "appuniversum/u-align-text";
-@import "appuniversum/u-background";
-@import "appuniversum/u-break-word";
-@import "appuniversum/u-headings";
-@import "appuniversum/u-hide";
-@import "appuniversum/u-flex";
-@import "appuniversum/u-font-family";
-@import "appuniversum/u-font-weights";
-@import "appuniversum/u-max-widths";
-@import "appuniversum/u-paragraphs";
-@import "appuniversum/u-print";
-@import "appuniversum/u-responsive-spacings";
-@import "appuniversum/u-spacings";
-@import "appuniversum/u-visible";
-@import "appuniversum/u-widths";
+@import "ember-appuniversum/a-settings";
```

Comment:
Silly question, but what system are we using for the file naming? What does the `a-` prefix mean here?

Reply:
It's based on ITCSS (https://appuniversum.github.io/appuniversum/guidelines/architecture/). The reason of the prefixes is to easily spot files and their purpose (also having them grouped alphabetically is quite handy). I used the "a" here for "all". "g" (group) was already taken by generic.